templates/azure-api-template-nodejs/swagger/swagger.json

```diff
@@ -0,0 +1,93 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "${domain}",
+    "version": "1.0"
+  },
+  "host": "${domain}.azurewebsites.net" ,
+  "basePath": "/api",
```

Comment:
Similar to https://github.com/tmobile/jazz/blob/master/templates/api-template-nodejs/swagger/swagger.json#L9, I believe this should contain $domain & path should contain $service (?)

Ex: /api/mydomain/myservice

Reply:
I am not sure what you mean by that, this is the proper template for azure. Do you mean just the names of the variables should be different?